Backend/UnitTesting/TestingUtils.cs

```diff
@@ -21,23 +21,30 @@ public byte[] GetRandomness()
         }
 
         public User CreateUserInDb()
+        {
+            
+            User u = new User
+            {
+                Email = Guid.NewGuid() + "@" + Guid.NewGuid() + ".com",
+                DateOfBirth = DateTime.UtcNow,
+                City = "Los Angeles",
+                State = "California",
+                Country = "United States",
+                PasswordHash = (Guid.NewGuid()).ToString(),
+                PasswordSalt = GetRandomness()
+            };
+
+            return CreateUserInDb(u);
+        }
+
+        public User CreateUserInDb(User user)
```

Comment:
This method doesn't add any functionality besides saving the user object, we should just add the DatabaseContext operation to the CreateUserInDb() method and remove the CreateUserInDb(User) method.

Reply:
What you described is how it was before, which doesn't allow me to modify the user before saving those changes. This approach allows me to do that.